yt_dlp/extractor/medici.py

```diff
@@ -1,67 +1,164 @@
+                'thumbnail': r're:^https?://medicitv-[abc]\.imgix\.net/movie/[^?]+\.jpg(?:\?[^?]+)?',
+                'description': 'md5:9411fe44c874bb10e9af288c65816e41',
+                'upload_date': '20200323',
+            },
+        },
+        {
+            'url': 'https://www.medici.tv/en/ballets/carmen-ballet-choregraphie-de-jiri-bubenicek-teatro-dellopera-di-roma',
+            'md5': '40f5e76cb701a97a6d7ba23b62c49990',
+            'expected_warnings': [r'preview'],
+            'info_dict': {
+                'id': '7857',
+                'ext': 'mp4',
+                'title': 'Carmen by Jiří Bubeníček after Roland Petit, music by Bizet, de Falla, Castelnuovo-Tedesco, and Bonolis',
+                'thumbnail': r're:^https?://medicitv-[abc]\.imgix\.net/movie/[^?]+\.jpg(?:\?[^?]+)?',
+                'description': 'md5:0f15a15611ed748020c769873e10a8bb',
+                'upload_date': '20240223',
+            },
+        },
+        {
+            'url': 'https://www.medici.tv/en/documentaries/la-sonnambula-liege-2023-documentaire',
+            'md5': '87ff198018ce79a34757ab0dd6f21080',
+            'expected_warnings': [r'preview'],
+            'info_dict': {
+                'id': '7513',
+                'ext': 'mp4',
+                'title': 'La Sonnambula',
+                'thumbnail': r're:^https?://medicitv-[abc]\.imgix\.net/movie/[^?]+\.jpg(?:\?[^?]+)?',
+                'description': 'md5:0caf9109a860fd50cd018df062a67f34',
+                'upload_date': '20231103',
+            },
+        },
+        {
+            'url': 'https://edu.medici.tv/en/masterclasses/yvonne-loriod-olivier-messiaen',
+            'md5': 'fb5dcec46d76ad20fbdbaabb01da191d',
+            'skip': 'The preview doesn\'t start from the start. Only works when authenticated.',
+            'info_dict': {
+                'id': '3024',
+                'ext': 'mp4',
+                'title': 'Olivier Messiaen and Yvonne Loriod, pianists and teachers',
+                'thumbnail': r're:^https?://medicitv-[abc]\.imgix\.net/movie/[^?]+\.jpg(?:\?[^?]+)?',
+                'description': 'md5:aab948e2f7690214b5c28896c83f1fc1',
+                'upload_date': '20150223',
+            },
+        },
+        {
+            'url': 'https://www.medici.tv/en/jazz/makaya-mccraven-la-rochelle',
+            'md5': '4cc279a8b06609782747c8f50beea2b3',
+            'expected_warnings': [r'preview'],
+            'info_dict': {
+                'id': '7922',
+                'ext': 'mp4',
+                'title': 'NEW: Makaya McCraven in La Rochelle',
+                'thumbnail': r're:^https?://medicitv-[abc]\.imgix\.net/movie/[^?]+\.jpg(?:\?[^?]+)?',
+                'description': 'md5:b5a8aaeb6993d8ccb18bde8abb8aa8d2',
+                'upload_date': '20231228',
+            },
+        },
+    ]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        # Sets csrftoken cookie
```

Comment:
I ask because it seems to be the only reason why the webpage was being downloaded -- we don't extract anything else from it. So if we still need to request the webpage, then we should leave the comment intact otherwise it's not obvious why we are doing it

Reply:
and if we don't need cookies set from the webpage request, then we should also remove the `self._download_webpage` call on the next line

Reply:
deleting `self._download_webpage` makes us fail auth, so I think csrftoken cookie is required. I'll restore the comment then

Reply:
sounds good. and can we change it to `_request_webpage()` instead?

Reply:
Okay. I changed it.